jupyter/README.md

```diff
@@ -0,0 +1,417 @@
+
+
+
+
+
+![alt_text](images/image4.png "image_tooltip")
+
+
+
+## Run interactively: "Hello Qubit" {#run-interactively-"hello-qubit"}
+
+To run interactively within the container, you can open a second shell window to the VM as you did [above](#open-a-ssh-window-to-your-vm).
+
+Now, find the container ID with docker ps:
+
+
+```
+    $ docker ps
+```
+
+
+
+![alt_text](images/image11.png "image_tooltip")
+
+
+The CONTAINER ID is a UID something like "79804d33f250". Now you can connect to the container:
+
+
+```
+    $ docker exec -it [CONTAINER_ID] /bin/bash
+```
+
+
+This shell is running inside the container, with Python3, cirq and qsim installed. Start python3:
+
+
+```
+	$ python3
+```
+
+
+You will see the output:
+
+
+```
+    Python 3.6.8 (default, Apr 16 2020, 01:36:27) 
+    [GCC 8.3.1 20191121 (Red Hat 8.3.1-5)] on linux
+    Type "help", "copyright", "credits" or "license" for more information.
+    >>>
+```
+
+
+You are now ready for an interactive session running qsim.
+
+
+### Build the circuit {#build-the-circuit}
+
+To import the libraries and build the circuit, copy and paste following into the VM window.
+
+
+```
+    import cirq
+    import qsimcirq
+
+    # Pick a qubit.
+    qubit = cirq.GridQubit(0, 0)
+
+    # Create a circuit
+    circuit = cirq.Circuit(
+       cirq.X(qubit)**0.5,  # Square root of NOT.
+    )
+    print("Circuit:")
+    print(circuit)
+```
+
+
+You should see the output as 
+
+```
+
+(0, 0): ───X^0.5───
+
+```
+
+
+### Run the circuit {#run-the-circuit}
+
+Now to see what the circuit does when under qsim. Again, copy the following into your VM window:
+
+
+```
+    simulator = qsimcirq.QSimSimulator()
+    result = simulator.simulate(circuit)
+    print("Result:")
+    print(result)
+```
+
+
+The output will be:
+
+
+```
+    measurements: (no measurements)
+    output vector: (0.5+0.5j)|0⟩ + (0.5-0.5j)|1⟩
+```
+
+
+You have successfully simulated a quantum circuit on Google Cloud Platform using a Singularity container.
+
+
+### Running your own script {#running-your-own-script}
+
+If you want to run a Python script, you can locate a file in the home directory on your VM, then run something like in the container shell
+
+	`$ python3 /homedir/myscript.py`
+
+
+### Exit the container {#exit-the-container}
+
+Exit the container by typing cntl-d twice. You will see the output like:
+
+
+```
+    [root@79804d33f250 /]# exit
+```
+
+
+-----
+
+
+## **Cleaning up** {#cleaning-up}
+
+_[The cleaning up section is required. Use it to ensure they don’t get billed for any extra time.]_
+
+To avoid incurring charges to your Google Cloud Platform account for the resources used in this tutorial:
+
+
+### Delete the VM {#delete-the-vm}
+
+It is usually recommended to delete the entire project you created for this tutorial, but if you want to continue using the project, you can easily either STOP the VM or Delete it. Select the VM, the either select the STOP square, or the DELETE trash can.
```

Comment:
"Delete" in "STOP the VM or Delete it" should either be "delete" or "DELETE", for consistency.